smart_contract/executor/src/default.rs

```diff
@@ -251,7 +251,7 @@ pub mod domain {
         authority: &AccountId,
         isi: &SetKeyValue<Domain>,
     ) {
-        let domain_id = isi.object_id();
+        let domain_id = isi.object();
```

Comment:
Although IMO it's totally fine to not rename local variables.

```suggestion
        let domain = isi.object();
```

Reply:
ah, I didn't go renaming local variables because that would sometimes conflict with entity names